mmv1/templates/terraform/examples/firebase_app_check_device_check_config_minimal.tf.erb

```diff
@@ -0,0 +1,29 @@
+resource "google_firebase_apple_app" "default" {
```

Comment:
Since the only difference is the TTL field, there's not much value (for tests or for docs) in having the minimal config in addition to the full config. You could leave this out if you want.

Reply:
Done.